src/it/scala/net/snowflake/spark/snowflake/SecuritySuite.scala

```diff
@@ -68,32 +69,50 @@ class SecuritySuite extends IntegrationSuiteBase {
     FileUtils.deleteQuietly(new File(TEST_LOG_FILE_NAME))
   }
 
+  ignore("manual test for addLog4j2FileAppender()/dropLog4j2FileAppender()") {
+    logger.info("Before adding file appender")
+    addLog4j2FileAppender(loggingFilePath, fileAppenderName)
+    logger.info("After adding file appender")
+    dropLog4j2FileAppender(fileAppenderName)
+    logger.info("After dropping file appender")
+  }
+
   test("verify pre-signed URL are not logged for read & write") {
     logger.info("Reconfigure to log into file")
     // Reconfigure log file to output all logging entries.
-    reconfigureLogFile(TEST_LOG4J_PROPERTY)
-
-    // Read from one snowflake table and write to another snowflake table
-    sparkSession
-      .sql("select * from test_table_large_result order by int_c")
-      .write
-      .format(SNOWFLAKE_SOURCE_NAME)
-      .options(thisConnectorOptionsNoTable)
-      .option("dbtable", test_table_write)
-      .mode(SaveMode.Overwrite)
-      .save()
-
-    // Check pre-signed is used for the test
-    assert(searchInLogFile(".*Spark Connector.*"))
-
-    // Check pre-signed URL are NOT printed in the log
-    // by searching the pre-signed URL domain name.
-    assert(!searchInLogFile(".*https?://.*amazonaws.com.*"))
-    assert(!searchInLogFile(".*https?://.*core.windows.net.*"))
-    assert(!searchInLogFile(".*https?://.*googleapis.com.*"))
+    if (USE_LOG4J2_PROPERTIES) {
+      addLog4j2FileAppender(loggingFilePath, fileAppenderName)
+    } else {
+      reconfigureLogFile(TEST_LOG4J_PROPERTY)
+    }
 
-    // Reconfigure back to the default log file.
-    reconfigureLogFile(DEFAULT_LOG4J_PROPERTY)
+    try {
+      // Read from one snowflake table and write to another snowflake table
```

Comment:
The test code is not actually changed. I just move it to `try {}` so that the logging reset can be executed even ifException is raised.